code/API_definitions/check-device-connectivity.yaml

```diff
@@ -181,6 +237,16 @@ components:
           $ref: '#/components/schemas/Port'
       required:
         - ueId
+    RequestConnectionStatus:
+      type: object
+      properties:
+        ueId:
+          $ref: '#/components/schemas/UeId'
+        eventType:
+          $ref: '#/components/schemas/NetworkConnectionEventType'
```

Comment:
Why is here an "eventType"?
It's not needed because you are just requesting with an UeId.